source_test.yaml

```diff
@@ -209,13 +233,14 @@
   type: 0
   ignore_patterns: ['PYSEC-0000.*', '^(?!PYSEC-).*$']
   directory_path: 'vulns'
-  repo_url: 'https://github.com/pypa/advisory-database.git'
+  repo_url: 'ssh://github.com/pypa/advisory-database'
```

Comment:
Should we use https for both prod and test? I'm not sure ssh creds exist in test... and they're not necessary in any case for this source.

Reply:
I'm happy to harmonise in that direction, I had biased towards what was in Production as the base.

Reply:
Done in 9256d75d